.buildkite/pipeline.yml

```diff
@@ -0,0 +1,18 @@
+steps:
+  - label: "Validate Clang Format"
+    command: "/code/NUbots/.travis/validate_format.sh"
```

Comment:
If we are going to transition we should probably swap over from travis to buildkite entirely

Reply:
I feel like that decision is above my paygrade 😜. But yeah we can easily switch to removing all references to Travis if we decide that.

Reply:
Well, if buildkite works and is 6 minutes to travis' 40 minutes we are going to use buildkite. There's no point waiting around for 40 minutes when we already have a CI done. We're going to use either buildkite or travis, but never both.

Reply:
Removed references to Travis except puppet stuff which is being sorted in #255.